scheduler/src/cook/mesos/api.clj

```diff
@@ -943,6 +984,15 @@
       false
       [true {::error "must supply at least one job or instance query param"}])))
 
+(defn job-request-malformed?
```

Comment:
We should stick to one protocol, above we are using `[true {::jobs ...}] or [false {::error ...}]`.